src/vrcosclib.Test/Utility/OscUtilityTests.cs

```diff
@@ -158,34 +162,40 @@ public void TestReceivePortOutOfRange(int port)
     [TestCase("127.0.0.1")]
     [TestCase("192.168.1.1")]
     [TestCase("8.8.8.8")]
-    public void TestClientIPAddress(string ipAddress)
+    public void TestVrcIPAddress(string ipAddress)
     {
-        string oldAddress = OscUtility.ClientIPAddress;
+        string oldAddress = OscUtility.VrcIPAddress;
+        var _oldClient = OscUtility.Client;
 
-        OscUtility.ClientIPAddress = ipAddress;
-        Assert.AreEqual(ipAddress, OscUtility.ClientIPAddress);
+        OscUtility.VrcIPAddress = ipAddress;
+        Assert.AreEqual(ipAddress, OscUtility.VrcIPAddress);
         Assert.AreEqual(ipAddress, OscUtility.Client.Destination.Address.ToString());
+        Assert.AreNotSame(_oldClient, OscUtility.Client);
 
-        OscUtility.ClientIPAddress = oldAddress;
+        OscUtility.VrcIPAddress = oldAddress;
     }
 
     [TestCase("example.com")]
     [TestCase("ipaddress")]
-    public void TestClientIPAddressInvalidFormat(string ipAddress)
+    public void TestVrcIPAddressInvalidFormat(string ipAddress)
     {
-        string oldAddress = OscUtility.ClientIPAddress;
-        Assert.Throws<FormatException>(() => OscUtility.ClientIPAddress = ipAddress);
-        Assert.AreEqual(oldAddress, OscUtility.ClientIPAddress);
-        Assert.AreEqual(oldAddress, OscUtility.Client.Destination.Address.ToString());
+        string oldAddress = OscUtility.VrcIPAddress;
+        var _oldClient = OscUtility.Client;
+
+        Assert.Throws<FormatException>(() => OscUtility.VrcIPAddress = ipAddress);
+        Assert.AreEqual(oldAddress, OscUtility.VrcIPAddress);
+        Assert.AreSame(_oldClient, OscUtility.Client);
     }
 
     [Test]
     public void TestClientIPAddressNull()
     {
-        string oldAddress = OscUtility.ClientIPAddress;
-        Assert.Throws<ArgumentNullException>(() => OscUtility.ClientIPAddress = null!);
-        Assert.AreEqual(oldAddress, OscUtility.ClientIPAddress);
-        Assert.AreEqual(oldAddress, OscUtility.Client.Destination.Address.ToString());
+        string oldAddress = OscUtility.VrcIPAddress;
+        var _oldClient = OscUtility.Client;
+
+        Assert.Throws<ArgumentNullException>(() => OscUtility.VrcIPAddress = null!);
+        Assert.AreEqual(oldAddress, OscUtility.VrcIPAddress);
+        Assert.AreSame(_oldClient, OscUtility.Client);
```

Comment:
Generation of new `OscUtility.Client` is not a requirement.
It seems better to make a test that actually communicates.

Reply:
This is to check that there is no side-effect when exceptions are thrown (everything is the same). Real communication test can be added for sure.

Reply:
oops, I made a mistake.
Verificationof new or not new OscUtility.Client is not a requirement. (I also point to other than this.)